exporter/clickhouseexporter/exporter_logs_test.go

```diff
@@ -101,6 +102,38 @@ func TestExporter_pushLogsData(t *testing.T) {
 	})
 }
 
+func TestLogsExporter_createDatabase(t *testing.T) {
+	initClickhouseTestServer(t, func(query string, values []driver.Value) error {
+		return nil
+	})
+	t.Run("tcp://mydatabase-clickhouse-headless:9000/mydatabase", func(t *testing.T) {
+		c := &Config{
+			TimeoutSettings: exporterhelper.NewDefaultTimeoutSettings(),
+			QueueSettings:   QueueSettings{QueueSize: exporterhelper.NewDefaultQueueSettings().QueueSize},
+			RetrySettings:   exporterhelper.NewDefaultRetrySettings(),
+			DSN:             "tcp://mydatabase-clickhouse-headless:9000/mydatabase",
+			LogsTableName:   "otel_logs",
+			TracesTableName: "otel_traces",
+			TTLDays:         7,
+		}
+		err := createDatabase(c)
+		require.NoError(t, err)
+	})
+	t.Run("tcp://newdatabase-clickhouse-headless:9000/newdatabase", func(t *testing.T) {
```

Comment:
This appears to test the same scenario as the test above, is there a difference between the two outside of the database name?

Reply:
done